pkgs/development/libraries/libdrm/default.nix

```diff
@@ -25,13 +28,15 @@ stdenv.mkDerivation rec {
   preConfigure = stdenv.lib.optionalString stdenv.isDarwin
     "echo : \\\${ac_cv_func_clock_gettime=\'yes\'} > config.cache";
 
-  configureFlags = [ "--enable-install-test-programs" ]
+  mesonFlags =
+    [ "-Dinstall-test-programs=true" ]
     ++ stdenv.lib.optionals (stdenv.isAarch32 || stdenv.isAarch64)
       [ "--enable-tegra-experimental-api" "--enable-etnaviv-experimental-api" ]
-    ++ stdenv.lib.optional stdenv.isDarwin "-C"
```

Comment:
please review this deletion. I'm not sure what it was for, or what it's meson equivalent is.

Reply:
It can be dropped as well, it's asking configure to use the cache ( https://www.gnu.org/software/autoconf/manual/autoconf-2.65/html_node/Cache-Files.html ), presumably to pick up the value put there in `preConfigure` previously.

Reply:
Not sure that the `clock_gettime` workaround is still useful (meson or no), may need some testing or investigation on that.